package.json

```diff
@@ -43,5 +43,9 @@
     "uglify-js": "2.4.x",
     "yui": "3.11.0",
     "yuidocjs": "0.3.x"
+  },
+  "devDependencies": {
```

Comment:
You should run npm strinkwrap so we can lock down these dependencies so they have less of a chance to break out from underneath us.